Build branches

paulmelnikow opened the pull request:
When merging someone else's PR with edits, it's helpful to to push a branch
and make sure the tests are also passing in CI.

Related: any reason not to discard these?

- old-master
- redundant-mit-license

espadrine commented:
> When merging someone else's PR with edits, it's helpful to to push a branch
> and make sure the tests are also passing in CI.

I'd like to avoid having many branches on the main repo.
If you don't trust a local `npm test`, could you set up Travis CI on your own repo?

(This comment is just informative, it doesn't preclude from merging this.)

> Related: any reason not to discard these?
>
> old-master
> redundant-mit-license

Not sure what redundant-blah is. old-master is the old code that got replaced with the current server software, back when we were at a two-digit star count, so we can probably remove it as well.

paulmelnikow commented:
Yeah… I've had too many experiences where things break at merge time, and I'd like to prevent that as much as possible.

> If you don't trust a local `npm test`, could you set up Travis CI on your own repo?

Sure, that's a good idea.

I removed `redundant-mit-license` and renamed `old-master` to `historical` for now.
